CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDU
 By participating, you are expected to uphold this code.
 Please report unacceptable behavior to [engineering@hellofresh.com](mailto:engineering@hellofresh.com).
```

Comment:
Should we keep their email here?

Reply:
Right, maybe not

Reply:
This is a fork, we should keep everything the same... including the namespaces